### PR TITLE
Knowledgemotion add getty publisher

### PR DESCRIPTION
**Pull request by williamberg**

### CODE REVIEW

File: src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionFileProcessor.java [hunk 1]
```diff
@@ -26,8 +26,9 @@
         KnowledgeMotionSourceConfig.from("BBC Worldwide", Publisher.KM_BBC_WORLDWIDE, "km-bbcWorldwide:%s", "http://bbc.knowledgemotion.com/%s"),
         KnowledgeMotionSourceConfig.from("British Movietone", Publisher.KM_MOVIETONE, "km-movietone:%s", "http://movietone.knowledgemotion.com/%s"),
         KnowledgeMotionSourceConfig.from("Bloomberg", Publisher.KM_BLOOMBERG, "bloomberg:%s", "http://bloomberg.com/%s"),
-        KnowledgeMotionSourceConfig.from("AP", Publisher.KM_AP, "ap:%s", "http://ap.knowledgemotion.com/%s")
-    );
+        KnowledgeMotionSourceConfig.from("AP", Publisher.KM_AP, "ap:%s", "http://ap.knowledgemotion.com/%s"),
+        KnowledgeMotionSourceConfig.from("Getty", Publisher.GETTY, "getty:%s", "http://gettyimages.co.uk/%s")
```
Comment (tsprlng) [inline src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionFileProcessor.java]: Basically you might want to prefix this like KM_GETTY and make up a URL pattern similar to the others – to reflect that this isn't actually Getty's full, own set of data, it's a subset that's specific to KM.

File: src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionSpreadsheetColumn.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
     DATE("Date"),
     DURATION("Duration"),
     KEYWORDS("Keywords"),
-    PRICE_CATEGORY("Price category (1= stock, 2=news, 3=brand)"),
```
Comment (tsprlng) [inline src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionSpreadsheetColumn.java]: they actually named a column this way?!

**Comment by tsprlng** [comment]
:+1: but with comments

File: src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionFileProcessor.java [hunk 3]
```diff
@@ -26,8 +26,9 @@
...
+        );
```
Comment (wwajerowicz) [inline src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionFileProcessor.java]: @williamberg  Shouldn't the Getty publisher be named KM_GETTY to be consistent with names of other KnowledgeMotion publisher?

File: src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionSpreadsheetColumn.java (hunk 2 above)
Comment (williamberg) [inline src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionSpreadsheetColumn.java]: lol bbq

File: src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionFileProcessor.java (hunk 1 above)
Comment (williamberg) [inline src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionFileProcessor.java]: We do actually want to add this to the existing publisher, that's been pretty well discussed.

File: src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionFileProcessor.java (hunk 3 above)
Comment (williamberg) [inline src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionFileProcessor.java]: https://github.com/atlasapi/atlas/pull/656#discussion_r28515041

File: src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionFileProcessor.java (hunk 1 above)
Comment (tsprlng) [inline src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionFileProcessor.java]: There is no existing publisher.

Comment (williamberg) [inline src/main/java/org/atlasapi/remotesite/knowledgemotion/KnowledgeMotionFileProcessor.java]: Ah. Then I agree with you.

**Pull request merged**